contracts/Stake/StakeManager.sol

```diff
@@ -94,9 +88,16 @@ contract StakeManager is Initializable, Ownable {
         super.initialize(_owner);
     }
 
+    // Only allow increment so previous stashes are not overwritten accidentally
+    // Only allow increment up to uint128, should be big enough for practical use cases (ref: ipv6)
+    // while making it practically impossible to overflow (would need 2^128 calls)
+    function incrementStashIndex(uint128 num) public onlyOwner {
```

Comment:
Why do we need this function ?

Reply:
Was thinking we might need to increment the index to account for previous registrations so it doesn't start at 0, but doesn't seem like its gonna be needed.

Reply:
Shall I remove this function then ?